internal/seclang/rule_parser.go

```diff
@@ -317,18 +329,23 @@ func ParseRule(options RuleOptions) (*corazawaf.Rule, error) {
 		rule:           corazawaf.NewRule(),
 		defaultActions: map[types.RulePhase][]ruleAction{},
 	}
-
-	defaultActions := options.Config.Get("rule_default_actions", []string{
-		defaultActionsPhase2,
```

Comment:
The `defaultActionsPhase2` constant defined as `const defaultActionsPhase2 = "phase:2,log,auditlog,pass"` was used as a default of defaultActions when no defaultActions were found. I feel like it is not really intuitive. This PR proposes to keep `defaultActionsPhase2` as the default for phase 2.
- If a SecDefaultAction of any other phase is defined it will not lead to delete `defaultActionsPhase2`
- A SecDefaultAction defined for phase:2 will override the `defaultActionsPhase2` (not triggered the double SecDefaultAction for one phase check just added)

Reply:
Agreed.

Reply:
@anuraaga because of this change, I did a small revert of a change you did [here](https://github.com/corazawaf/coraza/pull/721/files#diff-75b63bf7f2bafaf722595e631e506cf45c30a13833b8f48650b3e371a6f23741L22-R20). `defaultActionsPhase2` is now not assigned by default (where a slice assignation was handy), but only later on, if there are no other default actions at phase 2 defined by the user. It is just a string sent to the parser.